cloud4rpi/errors.py

```diff
@@ -30,8 +38,13 @@ class NotSupportedError(Exception):
     subprocess.CalledProcessError: 'Try run with sudo',
     InvalidTokenError:
         'Device token {0} is invalid. Please verify it.',
+    InvalidConfigError:
+        'Configuration is invalid. It must be an array.',
+    UnexpectedVariableTypeError:
+        'Not supported type for variable: {0}',
```

Comment:
It would be really helpful if we extend the error message with the actual supported types, e.g.
Not supported type for variable: something_invalid. Supported types: numeric, bool, string, etc. Something like that.

Reply:
What's the difference between `UnexpectedVariableTypeError` and `UnexpectedVariableValueTypeError` ?

Reply:
@vutenkov: full error message was:
'Unexpected value type for the "{0}" variable. It must be "bool", "numeric" or "string".'
@Himura2la : restored